credentials/safeChef/credential.json

```diff
@@ -13,7 +13,7 @@
     "name": "Jane Doe",
     "certification": {
       "type": "FoodSafetyCertification",
-      "certificateId": "urn:uuid:8460c404-c0a5-4653-aeae-44d03a1a7334",
+      "certificateNo": "urn:uuid:8460c404-c0a5-4653-aeae-44d03a1a7334",
```

Comment:
NOTE: I was getting this safe mode validation error - https://gist.github.com/JSAssassin/351aec9b95545ec32aa2aac0de8878de#file-safe-mode-validation-error-md when using `certificateId` here, noticed that in the context it is defined as `certificateNo`. Wasn't sure which one has to be changed, so for now I have updated `certificateId` to `certificateNo`. That fixed the issue. LMK if this is the correct fix or not.

Reply:
Using `certificateNo` is the correct fix.

Reply:
It should be `certificateId`: https://github.com/credential-handler/vc-examples/pull/1#discussion_r1003890622

Reply:
@dlongley So looks like this context needs to be updated then https://testing-json-ld.pages.dev/safechef-v1.context.jsonld. Do we control edits to the context or someone else will have to update this for us? cc: @gannan08 @BigBlueHat

Reply:
I don't know -- but we should be able to control edits to the context, I would think. I'm not sure what our model is around contexts for the examples we use. But if we create the examples and their contexts, I'd think we'd need to host those contexts to enable fixes over time. I doubt we can or want to guarantee that we will only ever use contexts that we don't control in the examples.